### DISCUSSION TITLE
ActionLink to new razor or module

### DISCUSSION THREAD

**Discussion by pnicosia80** [oqtane/oqtane.framework#3077] [Q&A] [answered]
Good morning everyone,
I created a new "test" razor in the module and I need to set an action link in index.razor that goes to the new "test" razor. How can I do?

What if the test.razor was in another module of the same project? 

Thanks in advance. 
Greetings

**Comment by sbwalker** [comment, marked as answer]
@adrianus80 the ActionLink component renders buttons which allow you to navigate to different components which are part of your module. So if your module has an Index.razor component and a  Test.razor component then you can include an ActionLink in the Index.razor such as:

```<ActionLink Action="Test" />``` 

The "Action" property references the razor component you want to load.

You will not be able to use the ActionLink to reference another module in the same project.
